docs/artifacts/agentsview-session-review.html

```
@@ -0,0 +1,81 @@
+<title>Reviewing what the agents did</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)